packages/frontend/web-ui/src/game/hooks/useGetMessageFinishedGame.ts

```diff
@@ -0,0 +1,52 @@
+import { models as apiModels } from '@cornie-js/api-models';
+import { useEffect, useState } from 'react';
+
+import { useGetUser, UseGetUserResult } from '../../user/hooks/useGetUser';
+
+export interface UseGetMessageFinishedGameResult {
+  messageFinishedGame: string | undefined;
+}
+
+export const useGetMessageFinishedGame = (
```

Comment:
```suggestion
export const useGetFinishedGameMessage = (
```

Why should we provide a message?

I understand this is not part of the state.

The problem to solve is printing a friendly error message in the view, it doensn't necesarily mean we must  provide this message from the hook. I understand the message can be built if the winner user is provided, why don't we provide a `winner: apiModels: UserV1` in the hook so the view can take it and build the message?

Reply:
Yesterday, I wanted to comment you that I could change the finished game message for a winner, because in the view of Finished Game it'd better to get a winner. I was just thinking about it, at night, when I started to program next components.

Reply:
I've changed the name to `useGetFinishedGameWinner`